test-dev/main.c

```diff
@@ -6,6 +6,8 @@
 #ifndef WIN32_LEAN_AND_MEAN
 #define WIN32_LEAN_AND_MEAN
 #endif
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x0400
```

Comment:
IMO: This second patch actually deserves applying independent of the vc project patch.

One thing of note: do we actually want to force `_WIN32_WINNT` to `0x0400`?
Wouldn't it be more appropriate if we set it to `0x0400` if it's not already set?
(Unless I'm missing something.)